### DISCUSSION TITLE
global styles throwing an error

### DISCUSSION THREAD

**Discussion by satelin2002** [chakra-ui/panda#766] [Q&A] [answered]
```
import { defineConfig, defineGlobalStyles } from "@pandacss/dev";

const globalCss = defineGlobalStyles({
  "html, body": {
    color: "gray.900",
    lineHeight: "1.5",
  },
});

export default defineConfig({
  globalCss,
  // Whether to use css reset
  preflight: true,```




```
 error ./src/styles/globals.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[3].oneOf[12].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[3].oneOf[12].use[3]!./src/styles/globals.css
Error: Build failed with 1 error:
panda.config.ts:3:0: ERROR: Transforming const to the configured target environment ("es5") is not supported yet
Import trace for requested module:
./src/styles/globals.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[3].oneOf[12].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[3].oneOf[12].use[3]!./src/styles/globals.css
./src/styles/globals.css
- warn Fast Refresh had to perform a full reload due to a runtime error.
```

**Comment by segunadebayo** [comment, marked as answer]
@satelin2002,

Update the target of your tsconfig `es6` or `esnext` to get it working.
